optionale.md

```diff
@@ -0,0 +1,106 @@
+#Cursuri optionale
+#Semestrul 1
+
+>> Dat fiind că fiecare student se poate înscrie la maxim 2 opționale într-un an, informațiile următoare cer ajutorul vostru pentru a fi actualizate sau corectare, acolo unde este cazul.
+>> Simțiți-vă liberi să adăugați opționale noi în listă sau să aduceți la zi cele deja completate, dacă profesorii sau modul de desfășurare / evaluare s-a modificat între timp.
+
+
+1. Aplicatii web dezvoltate cu Ruby on Rails (2017-2018)
+Profesor : 
```

Comment:
`Profesor` ajunge sa fie plasat pe linia anterioara (another one of markdown's inconveniences)

Reply:
Eu unul editez fisierele in `vim` si folosesc `visual studio code` ca sa vad rezultatul final, pentru ca are optiunea de live preview la cum arata fisierul.